python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -119,6 +120,22 @@ def get_all_configs(self, prefix=False):
                 "Couldn't read config from {} paths".format(fail_count))
         return self._configs
 
+    @staticmethod
+    def get_trial_checkpoints_paths(trial, metric=TRAINING_ITERATION):
+        """Returns a list of (path, metric) pairs for all disk checkpoints of
+         a trial.
+
+        Parameters:
+            trial: getting the checkpoints for a specific trial.
+            metric (str): key for trial info to return, e.g. "mean_accuracy".
+            "training_iteration" is used by default.
+        """
+        from ray.tune.checkpoint_manager import Checkpoint
+
+        checkpoints = trial.checkpoint_manager.best_checkpoints()
```

Comment:
this is odd because the user may not have access to the Trial object... maybe we can load the path instead?

Reply:
Oh I thought this can be the next step after get_best_trial or just using self.trials. We can load from disk check points given the trial logdir.

Shall we support both? i.e. trial can be a trial or a path, and we take different approaches according to its type.

Reply:
Also I assume this should be a static method that can operate without trials in memory

Reply:
Ah sorry for the slow reply - supporting both would be ideal. `get_best_trial` is only available for live experiments (not offline analysis).

Reply:
done